VVRestApi/VVRestApi/Common/Messaging/HttpHelper.cs

```diff
@@ -650,29 +650,61 @@ public static T PostMultiPart<T>(string virtualPath, string queryString, UrlPart
 
                 client.DefaultRequestHeaders.Add("Authorization", "Bearer " + apiTokens.AccessToken);
 
-                using (var multiPartContent = new MultipartFormDataContent("Upload----" + DateTime.Now.ToString(CultureInfo.InvariantCulture)))
```

Comment:
The boundry string was removed in BuildMultipartFormDataContent. Are we ok with this?

Reply:
Whoops, that was unintentional. Good catch!